src/Shell/Task/StageTask.php

```diff
@@ -378,7 +378,7 @@ protected function _makeDir($path)
     protected function _getPath()
     {
         if (empty($this->args[0]) || !file_exists($this->args[0])) {
-            throw new InternalErrorException('Path not specified or invalid.');
+            throw new Exception('Path not specified or invalid.');
```

Comment:
Could also be an InvalidArgumentException()?

Reply:
Yes, it is better suitable in this case.

I just didn't wanted use exception for  `HTTP 500` error in shell, so I just took generic CakePHP exception.